addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -2807,10 +2807,10 @@ QUnit.module("Views", (hooks) => {
         assert.verifySteps(["read1", "willStart"]);
 
         await click(target.querySelector(".o_form_statusbar button.p"));
-        assert.verifySteps(["read2", "willUpdateProps", "willUpdateProps"]);
+        assert.verifySteps(["willUpdateProps", "read2", "willUpdateProps", "willUpdateProps"]);
 
         await click(target.querySelector(".o_form_statusbar button.p"));
-        assert.verifySteps(["willUpdateProps", "read3", "willUpdateProps", "willUpdateProps"]);
+        assert.verifySteps(["willUpdateProps", "willUpdateProps", "read3", "willUpdateProps", "willUpdateProps"]);
```

Comment:
These changes are weird but it's a side effect of `await afterExecuteAction(clickParams);` in `view_button_hook` and the mocked action service in this test.
My guess it that [these line](https://github.com/odoo/odoo/blob/16.0/addons/web/static/src/views/view_button/view_button_hook.js#L89-L90) cause two different renders and that one of them is executed while awaiting `afterExecuteAction`.